internal/file/file.go

```diff
@@ -475,6 +475,43 @@ func ListInDir(path string) ([]string, error) {
 	return files, nil
 }
 
+func DeleteDir(ctx context.Context, path string) (err error) {
+	exists, _, err := ExistsWithDetail(path)
+	if !exists {
+		return err
+	}
```

Comment:
It seems to the error won't be returned when `exists=true` and `err != nil`.
I'm not sure it is right.

Reply:
Thank you for your feedback. I have made changes to the method to return nil and output a debug log when the path does not exist, in order to ensure that the method exits normally without performing the deletion.